website/content/docs/beta-features.md

```diff
@@ -120,3 +120,14 @@ Template tags produce the following output:
 - `{{collection}}`: the name of the collection containing the entry changed
 
 - `{{path}}`: the full path to the file changed
+
+## Custom CMS branch names to keep multiple deployments with branches and separate collection data versions. This allows one to configure what branch prefix the instance will use for unpublished entries.
+
+Example:
+
+```yaml
+backend:
+  cms_branch_prefix: cms-dev
+```
+
+The default for this value is "cms"
```

Comment:
Docs need to be cleaned up before merge.

Reply:
I can clean it up, but I am not sure what cleaning is required?

Reply:
Most of the content of the docs here is in the heading, which should just be a short title such as "Custom Branch Name Prefix". The content in the heading should be moved to a short explanatory paragraph, and the 2nd-person voice ("You can configure...", "To enable this feature, you can set...") from earlier in the document should be maintained. A quick note of the reason you might use this (i.e., branch name collisions) would be good too. Thanks!